README.md

```diff
@@ -6,7 +6,7 @@
 
 
 ## Dependencies
-- [Node](https://nodejs.org/en/download/)
+- [Node v10.16](https://nodejs.org/en/download/)
```

Comment:
```suggestion
- [Node](https://nodejs.org/en/download/)
```

I suggest removing this, otherwise we'll forget about it 😄

Reply:
I originally put it there because the README file implies that you need to download and install Node separately to yarn. In which case, it should be obvious in the README what version is required.

Does Node need to installed independently, or will yarn pull it in automatically?

Reply:
Fair enough, you are correct that node has to be installed first.

Let's leave it in for now then – @TrentHouliston can we fix `NUClearNet.js`?